CONTRIBUTING.md

```diff
@@ -89,7 +89,14 @@ We've added some `UnoCSS` utilities styles to help you with that:
 
 ## Internationalization
 
-We are using [vue-i18n](https://vue-i18n.intlify.dev/) via [nuxt-i18n](https://i18n.nuxtjs.org/) to handle internationalization.
+We are using [vue-i18n](https://vue-i18n.intlify.dev/) via [nuxt-i18n](https://v8.i18n.nuxtjs.org/) to handle internationalization.
+
+You can check current [translation status](https://elk-docs.netlify.app/docs/guide/contributing#translation-status): more instructions on table caption.
```

Comment:
Isn't `docs.elk.zone` ?

```suggestion
You can check current [translation status](https://docs/elk.zone/docs/guide/contributing#translation-status): more instructions on table caption.
```

Reply:
it is `docs.elk.zone`

Reply:
cannot apply your suggestion, the link is wrong ;)

Reply:
updated